homework8/task3.py

```diff
@@ -0,0 +1,18 @@
+class EqualSignContext:
+    def __init__(self):
+        self.sign = "="
+
+    def __enter__(self):
+        print(self.sign*10)
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if exc_val:
+            print(f"Error occured: {exc_val}")
+        return True
+
+a = EqualSignContext()
+with a:
+    print("This code will be executed")
+    x = 1 / 0
+    print("This code will not be executed")
+print(a.sign * 10)
```

Comment:
By `move this like outsite if block` I mean exactly moving outside a block but not outside the whole class. Move it back to `__exit__` method:
```
    def __exit__(self, exc_type, exc_val, exc_tb):
        if exc_val:
            print(f"Error occured: {exc_val}")
        print(a.sign * 10)
        return True

Reply:
I tried to do in such way, but it didn't work. Now, I've realised in what problem was. Pardon me)